forms/jqm/element.js

```diff
@@ -53,7 +53,10 @@ define(function (require) {
     },
 
     remove: function () {
-      this.$label = null;
+      if (this.$label) {
+        this.$label.remove();
+        this.$label = null;
```

Comment:
`.remove()` detaches the DOM element from the DOM. So the DOM element referred to by `this.$el` will no longer have `this.$label` as a descendant.